package.json

```diff
@@ -8,14 +8,8 @@
     "pnpm": "^8"
   },
   "scripts": {
-    "prebuild": "node tools/clean.js",
-    "build": "tsc && cp babel.config.json build/ && node tools/copy-dist-files.js",
```

Comment:
nit: for frontend developers you could replace the command with the bazel equivalent rather than delete the script altogether

Reply:
Good idea. Added.